React/src/pages/employees/helper.ts

```diff
@@ -84,4 +112,41 @@ export const filterInitStructure: Array<FilterInputType> = [
         inputType: 'text',
         typeValue: 'string',
     },
+    {
+        id: 'phone',
+        value: '',
+        label: 'Phone',
+        inputType: 'text',
+        typeValue: 'string',
+    },
+    {
+        id: 'pesel',
+        value: '',
+        label: 'Pesel',
+        inputType: 'text',
+        typeValue: 'string',
+    },
 ]
+
+export const emptyForm = {
+    id: null,
+    firstName: '',
+    lastName: '',
+    email: '',
+    password: '',
+    username: '',
+    status: '',
+    roles: [],
+    phone: '',
+    pesel: ''
+}
+
+export const validationSchema = yup.object({
+    firstName: yup.string().min(2, 'Imię musi zawierać co najmniej 2 znaki').required('Wprowadź imię'),
+    lastName: yup.string().min(2, 'Nazwisko musi zawierać co najmniej 2 znaki').required('Wprowadź nazwisko'),
+    password: yup.string().min(8, 'Hasło musi zawierać co najmniej 8 znaków').required('Wprowadź hasło'),
+    username: yup.string().min(3, 'Nazwa musi zawierać co najmniej 3 znaki').required('Wprowadź nazwe'),
+    email: yup.string().email('Wymagany jest poprawny email').required('Wprowadź email'),
+    phone: yup.string().min(9,'Wymagany jest poprawny numer telefonu').required('Wprowadź numer telefonu'),
+    pesel: yup.string().length(11,'Wymagany jest poprawny pesel').required('Wprowadź email')
```

Comment:
```suggestion
    pesel: yup.string().length(11,'Wymagany jest poprawny pesel').required('Wprowadź pesel')
```
Pozostałość przy kopiowaniu